data.sql

```diff
@@ -1,5 +1,35 @@
 /* Populate database with sample data. */
 
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES (  'Agumon', DATE '2020-02-03', '0', 'true', '10.23'    );
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Gabumon', DATE '2018-11-15', '2', 'true', '8.00');
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Pikachu', DATE '2021-01-07', '1', 'false', '15.04');
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Devimon', DATE '2017-05-12', '5', 'true', '11.00');
+
+-- day - 2
+
+INSERT INTO animals (name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Charmander', DATE '2020-02-08', '0', 'false', '11.00');
+
+INSERT INTO animals ( name, date_of_birth, escape_attempts, neutered, weight_kg)
+VALUES ('Plantmon', DATE '2021-11-15', '2', 'true', '5.70');
```

Comment:
- [ ] The weight here should be -5.7 and not 5.7.

Reply:
Corrected